dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+                        <h4 :id="subcategory.name" class="anchor title">{{ subcategory.label }}</h4>
+                        <div :class="`p-badge badge-${category.color}`">
+                          <span>{{ category.label.toUpperCase() }}</span>
+                        </div>
+                      </div>
+
+                      <div class="col-9 formFields">
+                        <FormKit v-for="field in subcategory.fields" :key="field" type="group" :name="subcategory.name">
+                          <component
+                            v-if="field.showIf(companyAssociatedGdvData.data)"
+                            :is="field.component"
+                            :label="field.label"
+                            :placeholder="field.placeholder"
+                            :description="field.description"
+                            :name="field.name"
+                            :options="field.options"
+                            :required="field.required"
+                            :validation="field.validation"
+                            :validation-label="field.validationLabel"
+                            :reportingPeriod="yearOfDataDate"
+                            :data-test="field.name"
+                            :unit="field.unit"
+                            @field-specific-documents-updated="
+                              updateDocumentList(`${category.name}.${subcategory.name}.${field.name}`, $event)
+                            "
+                            :ref="field.name"
+                          />
+                        </FormKit>
+                      </div>
+                    </div>
+                  </template>
+                </div>
+              </FormKit>
+            </FormKit>
+          </FormKit>
+        </div>
+        <SubmitSideBar>
+          <SubmitButton :formId="formId" />
+          <div v-if="postGdvDataProcessed">
+            <SuccessMessage v-if="uploadSucceded" :messageId="messageCounter" />
+            <FailMessage v-else :message="message" :messageId="messageCounter" />
+          </div>
+
+          <h4 id="topicTitles" class="title pt-3">On this page</h4>
+          <ul>
+            <li v-for="category in gdvDataModel" :key="category">
+              <ul>
+                <li v-for="subcategory in category.subcategories" :key="subcategory">
+                  <a
+                    v-if="subcategoryVisibility.get(subcategory) ?? true"
+                    @click="smoothScroll(`#${subcategory.name}`)"
+                    >{{ subcategory.label }}</a
+                  >
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </SubmitSideBar>
+      </div>
+    </template>
+  </Card>
+</template>
+<script lang="ts">
+import { FormKit } from "@formkit/vue";
+import { ApiClientProvider } from "@/services/ApiClients";
+import Card from "primevue/card";
+import { computed, defineComponent, inject } from "vue";
```

Comment:
Another example for this:
![image](https://github.com/d-fine/Dataland/assets/104447806/cf211b0c-2fe7-4abb-b7c0-237cf14612f2)

Reply:
Weitere Akkreditierungen has been fixed

Reply:
This does not seem to be framework related. I'll look into this.

Reply:
For the example of "Ratingbericht": The problem lies here `wrapDisplayValueWithDatapointInformation(
                formatYesNoValueForDatatable(dataset.allgemein?.rating?.ratingbericht?.value),
                "Ratingbericht",
                dataset.allgemein?.rating?.ratingbericht,
              )` in the view configuration.
The result of `formatYesNoValueForDatatable` is passed into the slot of the slot of the `DataPointWrapperDisplayComponent` which handles the display of the icon link and the table. However `formatYesNoValueForDatatable` always only returns a string display component. I don't see how this could have ever been displayed as a link before. However, this is not even the problem here. In "MultiLayerDataTableCell.vue", there is simply the test "No data provided" slotted in without being turned to a link. I don't see how this should have ever worked before.

Reply:
Using something like `yesNoDataPointValueGetterFactory` handles more case distinctions. But is right now only used in the converted view configs.

Reply:
Also for simple datapoints there should not be a window opening but only a download starting.